### PR TITLE
Build signalfx handler

### PR DESCRIPTION
**Pull request by squirrely**

### CODE REVIEW

File: src/fullerite/handler/signalfx.go [hunk 1]
```diff
@@ -1,39 +1,138 @@
...
+			log.Println("Problem parsing interval value", intervalStr)
+			s.interval = defaultInterval
+		} else {
+			s.interval = val
+		}
+	}
 }
 
 // Run send metrics in the channel to SignalFx.
 func (s SignalFx) Run() {
```
Comment (baris) [inline src/fullerite/handler/signalfx.go]: Most of what this does would be useful to other handlers too. Shall we extract parts of this to handler?

File: signalfx.proto [hunk 2]
```diff
@@ -0,0 +1,51 @@
+package com.signalfx.metrics.protobuf;
```
Comment (baris) [inline signalfx.proto]: Nice! Let's start using this.

File: src/fullerite/handler/handler.go [hunk 3]
```diff
@@ -33,11 +32,13 @@ func New(name string) Handler {
...
 	Run()
 	Configure(*map[string]string)
+
+	// taken care of by the base
 	Name() string
 	Channel() chan metric.Metric
 
-	Interval() int
-	SetInterval(int)
+	Interval() float64
```
Comment (baris) [inline src/fullerite/handler/handler.go]: Why is interval a float?

Comment (squirrely) [inline src/fullerite/handler/handler.go]: so if you want to do it every 1.5 seconds?

**Pull request merged**